cron/sqs_queue_status.py

```diff
@@ -100,11 +100,11 @@ def getQueueSizes():
         qcounter -= 1
     # post to elasticsearch servers directly without going through
     # message queues in case there is an availability issue
-    es.save_event(index=options.index, doc_type='mozdefhealth', body=json.dumps(healthlog))
+    es.save_event(index=options.index, body=json.dumps(healthlog))
```

Comment:
We need to modify the event before these lines to set the type = 'mozdefhealth'

Reply:
Fixed in https://github.com/mozilla/MozDef/pull/1114/commits/bf3a143b0c9b5058fcc6221d776e2724a2d5d7ac